<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>:/resource/Text.icon</activeon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white; background-color:#051B2B;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="CodeEditor" name="textEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="findEdit">
      <property name="styleSheet">
       <string notr="true">background-color:white;color:black;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menu">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>файл</string>
    </property>
    <addaction name="create"/>
    <addaction name="open"/>
    <addaction name="save"/>
    <addaction name="save_as"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>инструменты</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>поиск</string>
     </property>
     <addaction name="findText"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="setting"/>
    <addaction name="menu_4"/>
    <addaction name="reset"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>справка</string>
    </property>
    <addaction name="aboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>вид</string>
    </property>
    <addaction name="up"/>
    <addaction name="down"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
  </widget>
  <action name="create">
   <property name="text">
    <string>создать новый файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="open">
   <property name="text">
    <string>открыть файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="save_as">
   <property name="text">
    <string>сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="setting">
   <property name="text">
    <string>параметры</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>выход</string>
   </property>
  </action>
  <action name="reset">
   <property name="text">
    <string>отмена</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="aboutQt">
   <property name="text">
    <string>О Qt...</string>
   </property>
  </action>
  <action name="findText">
   <property name="text">
    <string>поиск текста</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="up">
   <property name="text">
    <string>увеличить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="down">
   <property name="text">
    <string>уменьшить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
